dvc/exceptions.py

```diff
@@ -303,3 +303,13 @@ def __init__(self, hook_name):
             "https://man.dvc.org/install "
             "for more info.".format(hook_name)
         )
+
+
+class CheckoutFailedException(DvcException):
```

Comment:
It should be
```suggestion
class CheckoutFailedError(DvcException):
```
or even
```suggestion
class CheckoutError(DvcException):
```